.github/workflows/publish_packages.yml

```diff
@@ -0,0 +1,56 @@
+name: publish_packages.yml
+on:
+  release:
+    types:
+      - published
```

Comment:
I think `published` will also trigger on creating a draft release which our workflow does. I think it should be `released` so it only uploads to registry if we publish a new release.

Reply:
Also a concern, in case we ever create a pre-release, do we want to upload to registry as well? Does it support pre-release versions?

cc @maciektr

Reply:
You are right, it should be changed to `released`
https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=released#release

Reply:
Okay, if we want to upload pre-releases as well, let's keep it at published.